linux/AppImage/apprun-hooks/install-url-handler.sh

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env bash
+
+# If anything happens
+# notify the user that cloud functions may be restricted
+# but do not prevent from launching the app
+error_handler() {
+    local exit_code=$?
+    local line_number=$1
+    local command="$2"
+    echo "Error: Command '${command}' failed with exit code ${exit_code} at line ${line_number}."
+    echo "You may not be able to use some cloud functions"
```

Comment:
Reading this at first I understood it as "I am not able to use some cloud functions, and hence I'm getting this error", but in fact it's the other way round.
Maybe this particular line isn't useful? (Also it won't get translated.)

Reply:
@saintmatthieu  The message is shown (or visible) only if audacity is started from the command line. 
We can use `zenity` to pop up an error, but I don't want to be too intrusive. 
The reasoning is that in most cases if the user is getting this error, it's because they have a non-standard configuration, and nagging about this issue all the time would not be appropriate. Most likely there will be no solution in that case other than to use manual "copy the token" method, and even then the cloud interaction would be limited. For example opening the project from the browser window wouldn't work. So the message stays true "You may not be able to use some cloud functions". 

Also, when seeing the error about the cloud it lets the user know what they may be missing, and if they do not care about cloud, then they can just safely ignore it.

Reply:
We currently have no infrastructure for translating bash scripts, and I don't feel like it is required here. If anything, it would be more searchable if it were in English only.

Reply:
@saintmatthieu let me know if you want to provide a better wording for the error message